transports/sqs/operations-scripting_create_sqs_[5,).partial.md

```diff
@@ -132,4 +138,97 @@ sqs-transport endpoint subscribe [name] [event-type]
 
 `-r` | `--region`: Overrides the environment variable 'AWS_SECRET_ACCESS_KEY'
 
+`-p` | `--prefix`: Prefix to prepend to the topic provisioned for the event type and the subscribing queue
+
+### sqs-transport endpoint set-policy events
+
+Set the IAM policy on the input queue of an endpoint, based on the event types the endpoint subscribed to, using:
+
+```
+sqs-transport endpoint set-policy [name] events 
+                                         [--event-type]
+                                         [--access-key-id]
+                                         [--secret]
+                                         [--region]
+                                         [--prefix]
+```
+
+#### arguments
+
+`name` : Name of the endpoint (required)
+
+#### options
+
+`-evt` | `--event-type` : Full name of the event allow through the IAM policy (e.g. MyNamespace.MyMessage), can be repeated multiple times to allow multiple event types to pass.
```

Comment:
Could this simply be `-e`? Or is it that way to be aligned with the wildcard switches also having more than one letter?

Reply:
`-e` was already taken by `--expiration` on `large-message-support`